scripts/dist-git/setup_git_package

```diff
@@ -7,6 +7,7 @@ umask 0002
 # Figure out the environment we're running in
 eval "$(crudini --format=sh --get /etc/dist-git/dist-git.conf dist-git)"
 REPODIR=$gitroot_dir
+DEFAULT_MAIN=$default_branch
```

Comment:
Can you please also do the rename here? (DEFAULT_MAIN -> DEFAULT_BRANCH)

Reply:
+1 for DEFAULT_BRANCH

Reply:
@xsuchy can you take a look?  This IMO deserves fix.